app/code/Magento/GroupedProductGraphQl/etc/schema.graphqls

```diff
@@ -10,3 +10,21 @@ type GroupedProductItem @doc(description: "GroupedProductItem contains informati
     position: Int @doc(description: "The relative position of this item compared to the other group items")
     product: ProductInterface @doc(description: "The ProductInterface object, which contains details about this product option") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product")
 }
+
+type Mutation {
+    addGroupedProductsToCart(input: AddGroupedProductsToCartInput): AddGroupedProductsToCartOutput @resolver(class: "Magento\\GroupedProductGraphQl\\Model\\Resolver\\AddGroupedProductsToCart")
+}
+
+input AddGroupedProductsToCartInput {
+    cart_id: String!
+    cart_items: [GroupedProductCartItemInput!]!
+}
+
+input GroupedProductCartItemInput {
+    data: [CartItemInput!]!
+    parent_sku: String @doc(description: "Grouped product SKU.")
```

Comment:
```suggestion
```

Reply:
@lenaorobei not sure about this one. How do we add  grouped product without knowing it's sku or id?

Reply:
The idea was to add as `a group of simples`, so no parent sku is required. In shopping cart it will be listed as multiple simple products.

Reply:
I think we need to know which grouped product is being added. Same as with configurable - https://github.com/magento/graphql-ce/blob/2.3-develop/app/code/Magento/ConfigurableProductGraphQl/etc/schema.graphqls#L51